src/types/ReviewType.ts

```diff
@@ -1,10 +1,10 @@
 export interface Review {
```

Comment:
Maybe we should we have one convention on naming types? with  capital "i" prefix or without?